vagrant/provisioning/roles/foia-analytical-reports/tasks/main.yml

```diff
@@ -141,7 +141,7 @@
 - name: setup path to arkcase config file(Linux)
   become: yes
   become_user: pentaho
-  shell: sed -i 's~/home/marst/projects/ETL-projects/EEOC/foia---reports-dw-1/config/arkcase_config\.properties~{{ root_folder }}/install/pentaho/foia-reports-dw{{ foia_analytical_reports_version_formatted }}/config/arkcase_config\.properties~g' {{ root_folder }}/install/pentaho/foia-reports-dw{{ foia_analytical_reports_version_formatted }}/misc/VARIABLE_CONFIG.ktr
```

Comment:
On all these things, let's add an extra step, versus removing one, to maintain maximum compatibility with all report versions.  Like, here we could add a loop, and run two `sed` commands, one with EEOC and one without it.

Reply:
Or, just ansible lineinfile to replace whatever expression we give it, like "/home/marst/*" ?? That way, we will handle anything that is in the file, rather than separate EEOC and whatnot. And, it is ansible module, so...Give it a thought Dave, and i will change the code accordingly.

Reply:
👍 lineinfile, excellent idea.